vcr/api/v1/api_test.go

```diff
@@ -91,43 +89,16 @@ func TestWrapper_CreateDID(t *testing.T) {
 		assert.Error(t, err)
 	})
 
-	t.Run("error - validation error", func(t *testing.T) {
+	t.Run("error - issue error", func(t *testing.T) {
 		ctx := newMockContext(t)
 		defer ctx.ctrl.Finish()
 
 		ctx.echo.EXPECT().Bind(gomock.Any())
-		ctx.echo.EXPECT().String(http.StatusBadRequest, gomock.Any())
 		ctx.vcr.EXPECT().Issue(gomock.Any()).Return(nil, credential.ErrValidation)
 
 		err := ctx.client.Create(ctx.echo)
 
-		assert.NoError(t, err)
-	})
-
-	t.Run("error - DID not found", func(t *testing.T) {
-		ctx := newMockContext(t)
-		defer ctx.ctrl.Finish()
-
-		ctx.echo.EXPECT().Bind(gomock.Any())
-		ctx.echo.EXPECT().String(http.StatusBadRequest, gomock.Any())
-		ctx.vcr.EXPECT().Issue(gomock.Any()).Return(nil, fmt.Errorf("wrapped error: %w", types.ErrNotFound))
-
-		err := ctx.client.Create(ctx.echo)
-
-		assert.NoError(t, err)
-	})
-
-	t.Run("error - key not found", func(t *testing.T) {
-		ctx := newMockContext(t)
-		defer ctx.ctrl.Finish()
-
-		ctx.echo.EXPECT().Bind(gomock.Any())
-		ctx.echo.EXPECT().String(http.StatusBadRequest, gomock.Any())
-		ctx.vcr.EXPECT().Issue(gomock.Any()).Return(nil, fmt.Errorf("wrapped error: %w", types.ErrKeyNotFound))
-
-		err := ctx.client.Create(ctx.echo)
-
-		assert.NoError(t, err)
+		assert.ErrorIs(t, err, credential.ErrValidation)
```

Comment:
missing status code checks for this file.

Reply:
Ah missed those, good find